<ns0:test-suite xmlns:ns0="urn:model.allure.qatools.yandex.ru" start="1542911025474" stop="1542911045799">
  <name>test</name>
  <labels/>
  <test-cases>
    <test-case start="1542911025474" status="failed" stop="1542911031391">
      <name>test_add_group.test_add_group[None:::]</name>
      <failure>
        <message>AssertionError: assert [95:name Z9:h...gLhALTcL, ...] == [95:name Z9:he...gLhALTcL, ...]
  Left contains more items, first extra item: None:::
  Use -v to get the full diff</message>
        <stack-trace>app = &lt;fixture.application.Application object at 0x03DA0690&gt;
db = &lt;fixture.db.DbFixture object at 0x04265EB0&gt;, json_groups = None:::
check_ui = False

    def test_add_group(app, db, json_groups, check_ui):
        group = json_groups
        old_groups = db.get_group_list()
        app.group.create(group)
        time.sleep(2)
        new_groups = db.get_group_list()
        old_groups.append(group)
&gt;       assert sorted(old_groups, key=Group.id_or_max) == sorted(new_groups, key=Group.id_or_max)
E       assert [95:name Z9:h...gLhALTcL, ...] == [95:name Z9:he...gLhALTcL, ...]
E         Left contains more items, first extra item: None:::
E         Use -v to get the full diff

test\test_add_group.py:13: AssertionError</stack-trace>
      </failure>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="7496-MainThread"/>
        <label name="host" value="dondemon"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
    <test-case start="1542911031392" status="failed" stop="1542911034239">
      <name>test_add_group.test_add_group[None:name Z9:headeru:footeriiF VWjpYm]</name>
      <failure>
        <message>AssertionError: assert [95:name Z9:h...gLhALTcL, ...] == [95:name Z9:he...gLhALTcL, ...]
  Left contains more items, first extra item: None:name Z9:headeru:footeriiF VWjpYm
  Use -v to get the full diff</message>
        <stack-trace>app = &lt;fixture.application.Application object at 0x03DA0690&gt;
db = &lt;fixture.db.DbFixture object at 0x04265EB0&gt;
json_groups = None:name Z9:headeru:footeriiF VWjpYm, check_ui = False

    def test_add_group(app, db, json_groups, check_ui):
        group = json_groups
        old_groups = db.get_group_list()
        app.group.create(group)
        time.sleep(2)
        new_groups = db.get_group_list()
        old_groups.append(group)
&gt;       assert sorted(old_groups, key=Group.id_or_max) == sorted(new_groups, key=Group.id_or_max)
E       assert [95:name Z9:h...gLhALTcL, ...] == [95:name Z9:he...gLhALTcL, ...]
E         Left contains more items, first extra item: None:name Z9:headeru:footeriiF VWjpYm
E         Use -v to get the full diff

test\test_add_group.py:13: AssertionError</stack-trace>
      </failure>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="7496-MainThread"/>
        <label name="host" value="dondemon"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
    <test-case start="1542911034240" status="failed" stop="1542911037097">
      <name>test_add_group.test_add_group[None:name hKf 4Kcf:headerRcC TIsPCm:footer vFcw dAHgLhALTcL]</name>
      <failure>
        <message>AssertionError: assert [95:name Z9:h...gLhALTcL, ...] == [95:name Z9:he...gLhALTcL, ...]
  Left contains more items, first extra item: None:name hKf 4Kcf:headerRcC TIsPCm:footer vFcw dAHgLhALTcL
  Use -v to get the full diff</message>
        <stack-trace>app = &lt;fixture.application.Application object at 0x03DA0690&gt;
db = &lt;fixture.db.DbFixture object at 0x04265EB0&gt;
json_groups = None:name hKf 4Kcf:headerRcC TIsPCm:footer vFcw dAHgLhALTcL
check_ui = False

    def test_add_group(app, db, json_groups, check_ui):
        group = json_groups
        old_groups = db.get_group_list()
        app.group.create(group)
        time.sleep(2)
        new_groups = db.get_group_list()
        old_groups.append(group)
&gt;       assert sorted(old_groups, key=Group.id_or_max) == sorted(new_groups, key=Group.id_or_max)
E       assert [95:name Z9:h...gLhALTcL, ...] == [95:name Z9:he...gLhALTcL, ...]
E         Left contains more items, first extra item: None:name hKf 4Kcf:headerRcC TIsPCm:footer vFcw dAHgLhALTcL
E         Use -v to get the full diff

test\test_add_group.py:13: AssertionError</stack-trace>
      </failure>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="7496-MainThread"/>
        <label name="host" value="dondemon"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
    <test-case start="1542911037098" status="failed" stop="1542911039987">
      <name>test_add_group.test_add_group[None:nameiB mB :headerGS k1Re6 8Cvk:footerEX F]</name>
      <failure>
        <message>AssertionError: assert [95:name Z9:h...gLhALTcL, ...] == [95:name Z9:he...gLhALTcL, ...]
  Left contains more items, first extra item: None:nameiB mB :headerGS k1Re6 8Cvk:footerEX F
  Use -v to get the full diff</message>
        <stack-trace>app = &lt;fixture.application.Application object at 0x03DA0690&gt;
db = &lt;fixture.db.DbFixture object at 0x04265EB0&gt;
json_groups = None:nameiB mB :headerGS k1Re6 8Cvk:footerEX F, check_ui = False

    def test_add_group(app, db, json_groups, check_ui):
        group = json_groups
        old_groups = db.get_group_list()
        app.group.create(group)
        time.sleep(2)
        new_groups = db.get_group_list()
        old_groups.append(group)
&gt;       assert sorted(old_groups, key=Group.id_or_max) == sorted(new_groups, key=Group.id_or_max)
E       assert [95:name Z9:h...gLhALTcL, ...] == [95:name Z9:he...gLhALTcL, ...]
E         Left contains more items, first extra item: None:nameiB mB :headerGS k1Re6 8Cvk:footerEX F
E         Use -v to get the full diff

test\test_add_group.py:13: AssertionError</stack-trace>
      </failure>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="7496-MainThread"/>
        <label name="host" value="dondemon"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
    <test-case start="1542911039989" status="failed" stop="1542911042910">
      <name>test_add_group.test_add_group[None:namem:header8oyrL6L2t:footer6 Q29  Z WdCk ]</name>
      <failure>
        <message>AssertionError: assert [95:name Z9:h...gLhALTcL, ...] == [95:name Z9:he...gLhALTcL, ...]
  Left contains more items, first extra item: None:namem:header8oyrL6L2t:footer6 Q29  Z WdCk 
  Use -v to get the full diff</message>
        <stack-trace>app = &lt;fixture.application.Application object at 0x03DA0690&gt;
db = &lt;fixture.db.DbFixture object at 0x04265EB0&gt;
json_groups = None:namem:header8oyrL6L2t:footer6 Q29  Z WdCk , check_ui = False

    def test_add_group(app, db, json_groups, check_ui):
        group = json_groups
        old_groups = db.get_group_list()
        app.group.create(group)
        time.sleep(2)
        new_groups = db.get_group_list()
        old_groups.append(group)
&gt;       assert sorted(old_groups, key=Group.id_or_max) == sorted(new_groups, key=Group.id_or_max)
E       assert [95:name Z9:h...gLhALTcL, ...] == [95:name Z9:he...gLhALTcL, ...]
E         Left contains more items, first extra item: None:namem:header8oyrL6L2t:footer6 Q29  Z WdCk 
E         Use -v to get the full diff

test\test_add_group.py:13: AssertionError</stack-trace>
      </failure>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="7496-MainThread"/>
        <label name="host" value="dondemon"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
    <test-case start="1542911042912" status="failed" stop="1542911045799">
      <name>test_add_group.test_add_group[None:nameJ6 WYh:header Ww :footerej7]</name>
      <failure>
        <message>AssertionError: assert [95:name Z9:h...gLhALTcL, ...] == [95:name Z9:he...gLhALTcL, ...]
  Left contains more items, first extra item: None:nameJ6 WYh:header Ww :footerej7
  Use -v to get the full diff</message>
        <stack-trace>app = &lt;fixture.application.Application object at 0x03DA0690&gt;
db = &lt;fixture.db.DbFixture object at 0x04265EB0&gt;
json_groups = None:nameJ6 WYh:header Ww :footerej7, check_ui = False

    def test_add_group(app, db, json_groups, check_ui):
        group = json_groups
        old_groups = db.get_group_list()
        app.group.create(group)
        time.sleep(2)
        new_groups = db.get_group_list()
        old_groups.append(group)
&gt;       assert sorted(old_groups, key=Group.id_or_max) == sorted(new_groups, key=Group.id_or_max)
E       assert [95:name Z9:h...gLhALTcL, ...] == [95:name Z9:he...gLhALTcL, ...]
E         Left contains more items, first extra item: None:nameJ6 WYh:header Ww :footerej7
E         Use -v to get the full diff

test\test_add_group.py:13: AssertionError</stack-trace>
      </failure>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="7496-MainThread"/>
        <label name="host" value="dondemon"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
  </test-cases>
</ns0:test-suite>
